### PR TITLE
Add V4 hardware support

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Meadow.Clima/ClimaHardwareV4.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        /// </summary>
+        /// <param name="device">The meadow device</param>
+        /// <param name="i2cBus">The I2C bus</param>
+        public ClimaHardwareV4(IF7CoreComputeMeadowDevice device, II2cBus i2cBus)
+        : base(device, i2cBus)
+        {
+            _device = device;
+        }
+
+        internal override I2cConnector CreateQwiicConnector()
```
Comment (ctacke) [inline Source/Meadow.Clima/ClimaHardwareV4.cs:26]: base returns nullable....

File: Source/Meadow.Clima/IClimaHardware.cs [hunk 2]
```diff
@@ -78,6 +78,11 @@ public interface IClimaHardware
         /// </summary>
         public IRgbPwmLed? RgbLed { get; }
 
+        /// <summary>
+        /// Gets the Qwiic connector on the Clima board.
+        /// </summary>
+        public I2cConnector Qwiic { get; }
```
Comment (ctacke) [inline Source/Meadow.Clima/IClimaHardware.cs:84]: Should this be nullable?

File: Source/Meadow.Clima/ClimaHardwareV4.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/Meadow.Clima/ClimaHardwareV4.cs:26]: fixed

File: Source/Meadow.Clima/IClimaHardware.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/Meadow.Clima/IClimaHardware.cs]: fixed

**Pull request merged**